controller/controller.go

```diff
@@ -180,6 +204,11 @@ func WebStoreUpdateExtension(w http.ResponseWriter, r *http.Request) {
 
 // UpdateExtensions is the handler for updating extensions
 func UpdateExtensions(w http.ResponseWriter, r *http.Request) {
+	var data []byte
+	var err error
+
+	jsonPrefix := []byte(")]}'\n")
```

Comment:
Details about the prefix here: https://cs.chromium.org/chromium/src/components/update_client/protocol_parser_json.cc?q=jsonPrefix&dr=C&l=279